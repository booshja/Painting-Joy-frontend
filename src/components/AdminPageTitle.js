// dependencies
import React from "react";
import styled from "styled-components";

const StyledAdminPageTitle = styled.h1`
    font-family: "Barlow Condensed", sans-serif;
    font-size: 2.2rem;
    letter-spacing: 1px;
    padding: 1rem 1rem;
`;

const AdminPageTitle = ({ children }) => {
    return <StyledAdminPageTitle>{children}</StyledAdminPageTitle>;
};

export default AdminPageTitle;
