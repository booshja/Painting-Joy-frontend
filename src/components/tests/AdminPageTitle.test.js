import { render } from "@testing-library/react";
import AdminPageTitle from "../AdminPageTitle";

/** Smoke Test */
it("renders without crashing", () => {
    render(<AdminPageTitle>Homepage</AdminPageTitle>);
});

/** Snapshot Test */
it("matches snapshot", () => {
    const { asFragment } = render(<AdminPageTitle>Homepage</AdminPageTitle>);
    expect(asFragment()).toMatchSnapshot();
});

/** RTL Tests */
it("renders correctly", () => {
    const { getByText } = render(<AdminPageTitle>Homepage</AdminPageTitle>);

    getByText("Homepage");
});
