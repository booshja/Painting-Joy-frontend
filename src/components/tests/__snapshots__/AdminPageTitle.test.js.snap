// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  .c0 {
  font-family: "Barlow Condensed",sans-serif;
  font-size: 2.2rem;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  padding: 1rem 1rem;
  -webkit-align-self: flex-start;
  -ms-flex-item-align: start;
  align-self: flex-start;
}

<h1
    class="c0"
  >
    Homepage
  </h1>
</DocumentFragment>
`;
